packages/epo-widget-lib/src/charts/Tooltip/index.tsx

```diff
@@ -0,0 +1,54 @@
+import { FunctionComponent } from "react";
+import * as Styled from "./styles";
+
+export interface TooltipProps {
+  value: number | string;
+  x: number;
+  y: number;
+  height?: number;
+  className?: string;
+}
+
+const Tooltip: FunctionComponent<TooltipProps> = ({
```

Comment:
this tooltip differs a little from the one in rubin-obs-client, which is tied to individual points and bars, it is intended to be the only tooltip included in a chart and shown or hidden using a chart-level state. There is no formatter function in this component since it should be receiving it's data straight from the parent chart.